code/game/objects/items/cards_ids.dm

```diff
@@ -111,8 +113,8 @@
 	registered_account.replaceable = TRUE
 
 	// Applying the trim updates the label and icon, so don't do this twice.
-	if(ispath(trim))
-		SSid_access.apply_trim_to_card(src, trim)
+	if(ispath(trim_path) && !trim)
+		SSid_access.apply_trim_to_card(src, trim_path)
```

Comment:
what does this have to do with multiz

Reply:
https://github.com/LemonInTheDark/tgstation/commit/644813af784dc9b5a900e2d4d3f6dfad176fc54f

Reply:
Caused runtimes in init, and is also just annoying